_data/tagging-status.yml

```diff
@@ -133,6 +133,12 @@
    issues:
    updated: 2024-07-06
 
+ - name: axessibility
+   type: package
+   status: no-support
+   comments: loads tagpdf but it is not compatible with the current version. Its patches 
+             clash with the math module. phase-III+math should be used instead.
```

Comment:
suggestion:

... which provides broader tagging support without the need to load an additional package.